### PR TITLE
improve frontend login dialog

### PR DESCRIPTION
**Pull request by fbergkemper**

### CODE REVIEW

File: webui/module/Auth/src/Auth/Controller/AuthController.php [hunk 1]
```diff
@@ -253,7 +259,7 @@ public function getDirectorModel()
     *
     * @return object
     */
-   private function createNewLoginForm($form, $err_msg = null, $bsock = null)
+   private function createNewLoginForm($form, $multi_dird_env = null, $err_msg = null, $bsock = null)
```
Comment (franku) [inline webui/module/Auth/src/Auth/Controller/AuthController.php:262]:
```suggestion
   private function createNewLoginForm($form, $multi_dird_env, $err_msg = null, $bsock = null)
```

File: webui/module/Auth/view/auth/auth/login.phtml [hunk 2]
```diff
@@ -58,7 +58,11 @@ $this->headTitle($title);
          echo '<div class="input-group">';
       }
       echo '<div class="input-group-addon"><span class="glyphicon glyphicon-log-in" aria-hidden="true"></span></div>';
-      echo $this->formSelect($form->get('director')->setAttribute('class','form-control selectpicker'));
+      if($multi_dird_env) {
```
Comment (franku) [inline webui/module/Auth/view/auth/auth/login.phtml:61]: multi_dird_env: From which class is this variable?

Comment (fbergkemper) [reply webui/module/Auth/view/auth/auth/login.phtml:61]: It is passed into the View from the AuthController, see line 217.

File: webui/module/Auth/src/Auth/Controller/AuthController.php (hunk 1 above)
Comment (fbergkemper) [reply webui/module/Auth/src/Auth/Controller/AuthController.php:262]: It is passed to the View and checked there, I would like to leave it as it is.

**Pull request merged**